platforms/vercel-node-builder/vercel.json

```diff
@@ -8,5 +8,5 @@
       "VERCEL_NODE_BUILDER_PG_URL": "@vercel_node_builder_pg_url"
     }
   },
-  "builds": [{ "src": "*.js", "use": "@now/node" }]
+  "rewrites": [{ "source": "(.*)", "destination": "/api/app" }]
```

Comment:
@styfle In this test we wanted to test `@now/node` builder specifically, there was an issue with the builder while the `api` folder approach to create an API still worked.

Reply:
I see. I updated to the correct `@vercel/node` package.

Reply:
Thanks, and probably this would need to be reverted too, right?

`platforms/vercel-node-builder/index.js → platforms/vercel-node-builder/api/app.js`

Reply:
Yep, done 👍